### PR TITLE
Bunch of changes for correct dbe installing and conda packaging

### PR DESCRIPTION
**Pull request by Garra1980**

### CODE REVIEW

File: scripts/conda/build.sh [hunk 1]
```diff
@@ -13,7 +13,7 @@ export CXXFLAGS="`echo $CXXFLAGS | sed 's/-fPIC//'`"
 # (IIRC, it is needed for CUDA support):
 export CGO_ENABLED=1
 export CGO_LDFLAGS=
-export CGO_CFLAGS=$CFLAGS
+#export CGO_CFLAGS=$CFLAGS
```
Comment (anton-malakhov) [inline scripts/conda/build.sh:16]: ??

File: scripts/conda/build.sh [hunk 2]
```diff
@@ -65,23 +68,21 @@ cmake -S . -B build -Wno-dev \
     -DENABLE_PROFILER=OFF\
     -DENABLE_TESTS=OFF\
     -DENABLE_FSI=ON\
-    -DENABLE_TBB=ON\
-    -DENABLE_DBE=ON\
-    || exit 1
+    -DENABLE_DBE=ON
```
Comment (anton-malakhov) [inline scripts/conda/build.sh:71]: tbb is needed for FSI

File: scripts/conda/build.sh [hunk 3]
```diff
@@ -65,23 +68,21 @@ cmake -S . -B build -Wno-dev \
...
-# copy initdb to mapd_initdb to avoid conflict with psql initdb
-mv $PREFIX/bin/initdb $PREFIX/bin/omnisci_initdb
 cd ..
-rm -rf data
-mkdir data
+# copy initdb to mapd_initdb to avoid conflict with psql initdb
+#mv $PREFIX/bin/initdb $PREFIX/bin/omnisci_initdb
+#cd ..
+#rm -rf data
+#mkdir data
```
Comment (anton-malakhov) [inline scripts/conda/build.sh:83]: uncomment?

Comment (anton-malakhov) [reply scripts/conda/build.sh:83]: mv initdb --> omnisci_initdb should be uncommented

File: scripts/conda/build.sh (hunk 2 above)
Comment (anton-malakhov) [reply scripts/conda/build.sh:71]:
```suggestion
    -DENABLE_DBE=ON\
    -DENABLE_TBB=ON\    
```

File: scripts/conda/build.sh (hunk 3 above)
Comment (Garra1980) [reply scripts/conda/build.sh]: Why is this needed?

File: scripts/conda/build.sh (hunk 2 above)
Comment (Garra1980) [reply scripts/conda/build.sh]: I don't see ENABLE_TBB in CMakeLists - how is this supposed to work?

File: scripts/conda/build.sh (hunk 1 above)
Comment (Garra1980) [reply scripts/conda/build.sh]: Why we need this at all?

File: scripts/conda/build.sh [hunk 4]
```diff
@@ -1,87 +1,138 @@
...
-    export MACOSX_DEPLOYMENT_TARGET=10.12
+# Fixes https://github.com/Quansight/pearu-sandbox/issues/7
+#       https://github.com/omnisci/omniscidb/issues/374
+export CXXFLAGS="$CXXFLAGS -Dsecure_getenv=getenv"
+
+# Fixes `error: expected ')' before 'PRIxPTR'`
+export CXXFLAGS="$CXXFLAGS -D__STDC_FORMAT_MACROS"
+
+# Remove --as-needed to resolve undefined reference to `__vdso_clock_gettime@GLIBC_PRIVATE'
+export LDFLAGS="`echo $LDFLAGS | sed 's/-Wl,--as-needed//'`"
```
Comment (anton-malakhov) [inline scripts/conda/build.sh:27]: these changes are already in CMakeList.txt

File: scripts/conda/build.sh (hunk 3 above)
Comment (anton-malakhov) [reply scripts/conda/build.sh]: to get rid of conflicts for the name in global files namespace. They did have about it, please uncomment

File: scripts/conda/build.sh (hunk 2 above)
Comment (anton-malakhov) [reply scripts/conda/build.sh]: Probably on different branch? ok, please discard

**Pull request closed without merging**